azure-pipelines-dev.yml

```diff
@@ -236,8 +219,8 @@ stages:
 
           - script: |
               pip install --requirement requirements.txt
-              echo "about to run pip install $(GE_pytest_pip_opts)"
-              pip install $(GE_pytest_pip_opts)
+              echo "about to run pip install --requirement requirements-dev-test.txt --constraint constraints-dev.txt"
+              pip install --requirement requirements-dev-test.txt --constraint constraints-dev.txt
```

Comment:
Since this value is the same across all stages, let's remove the variable and just hard-code the value.